docs/api.rst

```diff
@@ -12408,3 +12408,36 @@ Get avatar for an ens name
    :statuscode 400: Invalid ens name provided (it doesn't end with `.eth`).
    :statuscode 404: There is no avatar set for the given ens name.
    :statuscode 409: Avatar was requested for an ens name that is not currently in the database or we couldn't query the blockchain.
+
+
+Clear Icons/Avatars Cache
+============================
+
+.. http:post:: /api/(version)/cache/images/<cache_type>/clear
+
+   Doing a POST on this endpoint will clear the cache of avatars/icons depending on the ``cache_type``. Valid options of ``cache_type`` are ``icons`` & ``avatars``.
+
+   **Example Request**
+
+    .. http:example:: curl wget httpie python-requests
+
+      POST /api/1/cache/images/icons/clear HTTP/1.1
+      Host: localhost:5042
+      Content-Type: application/json;charset=UTF-8
+
+      {
+         "entries": ["ETH", "BTC"]
+      }
+
+   :reqjsonarr optional[string] entries: An array of the items to be cleared from the cache. All items are deleted in the cache if ``null``.
+
+   **Example Response**:
+
+   .. sourcecode:: http
+
+      HTTP/1.1 200 OK
+      Content-Type: application/json
+
+   :statuscode 200: The cache items were cleared successfully.
+   :statuscode 400: Entries provided do not match the ``cache_type`` specified. Invalid ``cache_type`` provided.
+    :statuscode 500: Internal rotki error
```

Comment:
At least in github preview this seems like it has an extra space?
![2023-03-27_18-10](https://user-images.githubusercontent.com/1658405/228000135-ba6064c1-3f1e-49d8-9f2e-5869057dd972.png)
```suggestion
   :statuscode 500: Internal rotki error
```